lib/caesar.rb

```diff
@@ -1,6 +1,17 @@
-class Caesar
-  attr_reader :key
+class CaesarCipher
+  attr_accessor :data, :shift
+  def initialize(data, shift)
+    @data = data
+    @shift = 3
+  end
```

Comment:
It's good to see you have this pattern of passing values to initialize, setting attributes to those values, and giving them attr_accessors/readers/writers. However, this pattern isn't necessary for this challenge. Remember, initialize gets called when `new` is called, so if you look at the spec you'll see `caesar = Caesar.new`. There's no parameters in that new call, so your initialize method shouldn't have any parameters either. Also, in the spec the class is called `Caesar`, so your class should also be called Caesar.